xcode/Subconscious/Shared/Components/FirstRun/FirstRunDoneView.swift

```diff
@@ -10,20 +10,66 @@ import SwiftUI
 struct FirstRunDoneView: View {
     @ObservedObject var app: Store<AppModel>
     @Environment(\.colorScheme) var colorScheme
+    
+    private var status: ResourceStatus {
+        app.state.gatewayProvisioningStatus
+    }
+    
+    private var dottedLine: some View {
+        Line()
+            .stroke(style: StrokeStyle(
+                lineWidth: 3,
+                dash: [6, 3]
+            ))
+            .frame(height: 1)
+            .foregroundColor(.secondary.opacity(0.5))
+            .frame(width: 24)
+    }
+    
+    private var did: Did {
+        Did(app.state.sphereIdentity ?? "")
+            ?? Config.default.subconsciousGeistDid
+    }
 
     var body: some View {
         NavigationStack {
-            VStack {
+            VStack(spacing: AppTheme.padding * 4) {
```

Comment:
<img src="https://github.com/subconsciousnetwork/subconscious/assets/58421/f00d7617-e237-4028-a61b-48fb4dd687bb" width="500">

- Spacing should be equal on top and bottom of icon group.
- I think we could lose the cloud icon and just have the dotted line. Or perhaps the dotted line could be a dotted arrow?

Reply:
- Fixed spacing
- Changed to generic icons (no clouds)
- Removed accent color from icon in sync state

<img width="377" alt="Screenshot 2023-05-11 at 11 25 42 am" src="https://github.com/subconsciousnetwork/subconscious/assets/5009316/f3f4c2f0-7488-4f6a-aefd-96ebab574cab">